README.rst

```diff
@@ -61,6 +68,13 @@ These objects represent the standalone ``FiscalQuarter`` class.
    >>> b in a
    True
 
+You can also get the current ``FiscalQuarter`` with:
+
+.. code-block:: python
+
+   >>> from fiscalyear import *
+   >>> FiscalQuarter.current()
+   FiscalQuarter(2018, 2)
```

Comment:
Can you also add these same snippets to `docs/basic_usage.rst` in the same locations? Regrettably these files are almost duplicates of each other. I haven't found a good way to include one from the other yet. GitHub doesn't support rST includes 😢